services/wfm/src/main/java/org/openkilda/wfm/topology/cache/transport/CacheBolt.java

```diff
@@ -91,15 +90,17 @@
      */
     private InMemoryKeyValueState<String, Cache> state;
 
-    private CacheService cacheService;
+    private transient CacheService cacheService;
 
-    private final Auth pathComputerAuth;
+    private final PersistenceManager persistenceManager;
+
+    private transient RepositoryFactory repositoryFactory;
```

Comment:
It used only once. Why not get it from `persistenceManager` directly?

Reply:
Fixed.